regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -119,7 +119,7 @@ This contains all of the styles specific to the history
     .effective-label {
         padding: 0 15px;
         .font-regular;
-        color: @dark_gray;
+        color: #666666;
```

Comment:
Any reason not to stick to variables? Ideally we'd allow the pallet to change all at once.